designs/CentralNuGetManagement/CPVM-Project_Package_Version_Override.md

```diff
@@ -0,0 +1,425 @@
+# CPVM Project Package Version Override
+
+Users have indicated that Central Package Version Management (CPVM) needs a mechanism to override the centrally managed versions of packages on a project-by-project basis. Project Package Version Override allows individual projects to import different versions of NuGet packages organized in the same, centralized location.
+
+# Design Goals
+
+1. Even though package versions need to be specified for specific projects, keep the package versions centralized. Avoid situations where solution-level and project-level package version management are mixed in the same repository.
+2. Provide a means for organizing multiple package versions together.
+3. Manage the package versions of transitive dependencies alongside direct dependencies.
+4. Follow the MSBuild approach with minimal changes to existing XML patterns.
+
+# Feature Behavior
+
+- All `<PackageVersion>` should be defined within `Directory.Packages.props`, and `Version` may not be defined on `<PackageReference>` within projects.
+    - Specifying a `Version` on a `<PackageReference>` will throw a build error.
+    - Specifying a `<PackageVersion>` within a project file is an anti-pattern, and it will generate a build time warning. It will not throw an error, and a developer may choose to suppress the warning by specifying `<AllowProjectPackageVersion>true</AllowProjectPackageVersion>` within a `<PropertyGroup>`.
+- A `<PackageVersion>` may have a `CentralManagementGroup` defined which allows a `<PackageReference>` to refer to this version using the `CentralManagementGroup` attribute.
```

Comment:
`CentralManagementGroup` sounds odd. What about the original `VersionGroup` or `CentralVersionGroup`. I think just `Group` also makes sense in the context of NuGet/PackageReference.

Reply:
My suggestion would be to avoid the word "central" for this property as the feature itself is about overriding the "central" behavior :)